### PR TITLE
Feature/create contact database

### PR DESCRIPTION
**Pull request by XDRO**
## Describe your changes
Create a new model/company-contact-info.ts file to create table in keystone for company contact information when running the scripts `npm run generate` && `npm run dev`
Add export * from './company-contact-info.ts'; inside of index.ts file
## Issue ticket number and link
EES-8
https://tripleten-apiary.atlassian.net/browse/EES-8
## Checklist before requesting a review
image

- [ ] I have performed a self-review of my code
- [ ] If there are changes to custom resolvers, I have added / updated automated tests
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by michaelabudde — APPROVED** [review]
The schema for the CompanyContactInfo looks well-structured and works as expected. Approved. Nice work!

File: schema.graphql [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
-  lastLoginDate: DateTime
-}
-
-type PasswordState {
-  isSet: Boolean!
+  updatedAt: DateTime
+  status: String
 }
 
 scalar DateTime @specifiedBy(url: "https://datatracker.ietf.org/doc/html/rfc3339#section-5.6")
```
Comment (RPinkha) [inline schema.graphql:19]: This isn't even your code, so I am approving anyway, but
maybe it would be better to use the Javascript native method to get the date and time so we aren't dependent on an outside server.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

**Review by RPinkha — APPROVED** [review]
I don't see any issues. 
I am not certain about the fields. Maybe we can cut down on some, but tomer can comment on that as the task hasn't mentioned anything.

File: models/company-contact-info.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { list } from "@keystone-6/core";
+import { checkbox, text, timestamp, select } from "@keystone-6/core/fields";
+// import {relationship} from "@keystone-6/core/fields";
```
Comment (tomershvueli) [inline models/company-contact-info.ts]: Can we remove comments if they're no long necessary?

File: models/company-contact-info.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      db: { updatedAt: true },
+    }),
+    status: select({
+      options: [
+        { label: "Active", value: "active" },
+        { label: "Inactive", value: "inactive" },
+      ],
+      defaultValue: "active",
+    }),
+    // owner: relationship({
```
Comment (tomershvueli) [inline models/company-contact-info.ts]: Same here.

File: migrations/20240330041510_/migration.sql [hunk 4]
```diff
@@ -1,5 +1,5 @@
 -- CreateTable
-CREATE TABLE `User` (
+CREATE TABLE IF NOT EXISTS `User` (
```
Comment (tomershvueli) [inline migrations/20240330041510_/migration.sql]: Let's undo the changes to this file. What happens here is that Keystone will catch it and warn you the following: 
```
The migration `20240330041510_` was modified after it was applied.

We need to reset the mysql database "ees-db" at localhost:3306.
? Do you want to continue? All data will be lost › (Y/n)
```


We want to be able to run `npm run dev` from this branch without breaking changes to our DB.

File: package.json [hunk 5]
```diff
@@ -16,15 +16,16 @@
...
     "@keystone-6/fields-document": "^7.0.0",
     "@prisma/debug": "^5.3.1",
     "@prisma/generator-helper": "^5.3.1",
-    "dotenv": "^16.3.1",
+    "dotenv": "^16.4.5",
     "fs-extra": "^11.2.0",
     "keystone": "^4.2.1",
     "lodash": "^4.17.21",
-    "typescript": "^4.9.5"
+    "mysql2": "^3.11.0",
```
Comment (tomershvueli) [inline package.json]: Why did we bring in `mysql2` here? I don't think we need this, let's remove it.

**Comment by RPinkha** [comment]
@tomershvueli all issues were addressed. 
Ready for merge.

**Review by tomershvueli — APPROVED** [review]
Looks great!
